spec/dor/services/client/files_spec.rb

```diff
@@ -70,6 +70,7 @@
   describe '#preserved_content' do
     subject { client.preserved_content(filename: 'olemiss1v.jp2', version: 1) }
     before do
+      # allow(Deprecation).to receive(:warn) # it's probably better for this warning to display
```

Comment:
I think it's good to stub these in tests or the test output can become unreadable.

Reply:
all the other warnings are in sdr-services-app related classes that will go away;  this one is in a class that will stay, so I thought the single instance in the specs would be a good reminder.  I can take it out, tho.

Reply:
@jcoyne are you ok with the warning being shown until v4.0.0?

Reply:
I think this is a fine approach. Going to move forward with this for now, since @jcoyne is away and @ndushay has explained her rationale.